openslides_backend/action/actions/meeting/create_update_delete.py

```diff
@@ -200,29 +199,28 @@ def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
                 ],
             },
         ]
-        write_requests, action_results = self.execute_other_action(
-            GroupCreate, action_data
-        )
-
-        used_groups_dict = {
-            event["fields"]["name"]: event["fqid"].id  # type: ignore
-            for event in write_requests.events  # type: ignore
-            if event["type"] == EventType.Create
-            and event["fields"]["name"] in ("Admin", "Default")  # type: ignore
-        }
-        if len(used_groups_dict) != 2:
-            raise ActionException(
-                "There are no admin and default group created during meeting.create."
-            )
+        action_results = self.execute_other_action(GroupCreate, action_data)
 
-        instance["default_group_id"] = used_groups_dict["Default"]
-        instance["admin_group_id"] = used_groups_dict["Admin"]
+        fqid_default_group = FullQualifiedId(
+            Collection("group"), action_results[0]["id"]  # type: ignore
+        )
+        fqid_admin_group = FullQualifiedId(Collection("group"), action_results[1]["id"])  # type: ignore
+        assert (
+            self.datastore.additional_relation_models[fqid_default_group]["name"]
+            == "Default"
+        )
+        assert (
+            self.datastore.additional_relation_models[fqid_admin_group]["name"]
+            == "Admin"
+        )
+        instance["default_group_id"] = fqid_default_group.id
+        instance["admin_group_id"] = fqid_admin_group.id
 
         # Add user to admin group
         action_data = [
             {
                 "id": self.user_id,
-                "group_$_ids": {str(instance["id"]): [used_groups_dict["Admin"]]},
+                "group_$_ids": {str(instance["id"]): [fqid_admin_group.id]},
```

Comment:
You don't have to cast it to a string here, int should work as well. Just as a side note ;)

Reply:
The schema-validator don't accept it as int;-):
```
data = {'group_$_ids': {1: [3]}, 'id': 1}

>   ???
E   TypeError: expected string or bytes-like object

<string>:191: TypeError
```